tests/Fixer/Basic/BracesFixerTest.php

```diff
@@ -2259,8 +2259,8 @@ function D() /**
 //    if ($bar === \'bar\') {
 //        return [];
 //    }
-    
-    $bar = \'bar\';
+    '."\n".
```

Comment:
They help me learn that I should use `1 === preg_match('/\R$/', $previousToken->getContent())` instead of `$previousToken->isWhitespace("\n")`, so I keep them. :D
But I can remove them - not sure if they give any value or only noise.